packages/drivers/odsp-driver/src/odspUtils2.ts

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ITelemetryLogger } from "@fluidframework/common-definitions";
+import { fetchTokenErrorCode, throwOdspNetworkError } from "@fluidframework/odsp-doclib-utils";
+import {
+    IOdspResolvedUrl,
+    isTokenFromCache,
+    OdspResourceTokenFetchOptions,
+    TokenFetcher,
+    TokenFetchOptions,
+    tokenFromResponse,
+} from "@fluidframework/odsp-driver-definitions";
+import { PerformanceEvent } from "@fluidframework/telemetry-utils";
+import { IOdspSnapshot } from "./contracts";
+
+export function evalBlobsAndTrees(snapshot: IOdspSnapshot) {
+    let numTrees = 0;
+    let numBlobs = 0;
+    let encodedBlobsSize = 0;
+    let decodedBlobsSize = 0;
+    for (const tree of snapshot.trees) {
+        for(const treeEntry of tree.entries) {
```

Comment:
Nit: lack of space before `(`. BTW, does your repo complain about these sorts of things (in ours, I'm pretty sure `prettier` would have thrown an error).

Reply:
Done. We do have lint rules but perhaps this was not there.